Venmo/src/test/java/com/braintreepayments/api/venmo/MockkVenmoApiBuilder.kt

```diff
@@ -0,0 +1,90 @@
+package com.braintreepayments.api.venmo
+
+import io.mockk.every
+import io.mockk.mockk
+
+internal class MockkVenmoApiBuilder {
+
+    private var venmoPaymentContextId: String? = null
+    private var createNonceFromPaymentContextSuccess: VenmoAccountNonce? = null
+    private var vaultVenmoAccountNonceSuccess: VenmoAccountNonce? = null
+    private var createPaymentContextError: Exception? = null
+    private var createNonceFromPaymentContextError: Exception? = null
+    private var vaultVenmoAccountNonceError: Exception? = null
+
+    fun createPaymentContextSuccess(venmoPaymentContextId: String): MockkVenmoApiBuilder {
+        this.venmoPaymentContextId = venmoPaymentContextId
+        return this
+    }
+
+    fun createPaymentContextError(createPaymentContextError: Exception): MockkVenmoApiBuilder {
+        this.createPaymentContextError = createPaymentContextError
+        return this
+    }
+
+    fun createNonceFromPaymentContextSuccess(
+        createNonceFromPaymentContextSuccess: VenmoAccountNonce): MockkVenmoApiBuilder {
```

Comment:
nit:
```suggestion
        createNonceFromPaymentContextSuccess: VenmoAccountNonce
        ): MockkVenmoApiBuilder {
```

The convention when multiple lines of arguments are present is to put the closing parenthesis in the next line.